doc/developer-guides/c_coding_guidelines.rst

```diff
@@ -1107,7 +1107,7 @@ C-ST-01: The condition of a selection or iteration statement shall not be consta
 ==================================================================================
 
 The condition of a selection or iteration statement shall not be constant with
-the following exception, `do { ... } while (0)` shall be allowed if it is used
+the following exception: `do { ... } while (0)` shall be allowed if it is used
```

Comment:
Does the `do { ... } while (0)` render as code-block or should it be enclosed in double-backticks?

Reply:
Done.